_data/content/projects/en.yml

```diff
@@ -62,7 +62,7 @@ page_data:
       post: |
         Elisa was invited by the Mainel Foundation to present and take part in a panel of sector leaders discussing the role of technologies and experiences in exhibition-making. She focussed on various typologies she has worked on, from the immersive blockbuster exhibitions of the V&A Museum, London (David Bowie Is, ‘You Say You Want a Revolution?’ and Pink Floyd: Their Mortal Remains), World Expos (particularly Dubai Expo 2020) and technology-based experiences. 
         The Contemporary Visual Culture seminar series is part of the Mainel’s offering to the general public focussed on the improvement of society through solidarity and culture as transformative tools, to ensure that people are the protagonists of their own development. In her presentation, Elisa showed how immersive experiences are key to the democratisation of culture, as they turn passive visitors into active participants, whose understanding and feelings are just as important as the knowledge the curators are sharing. 
-        Both the topics of this discussion and the motivations behind the Mainel’s programming show a profound care for the public, whose lives both seek to enrich and empower. See <a href="https://mainel.org/noticias/expografias-coloquio-cvc-27/" target="blank"> Nuevas expografías: se puede interactuar con la obra incluso sin que esté físicamente presente · Fundación Mainel %</a>
+        Both the topics of this discussion and the motivations behind the Mainel’s programming show a profound care for the public, whose lives both seek to enrich and empower. See <a href="https://mainel.org/noticias/expografias-coloquio-cvc-27/" target="blank"> Nuevas expografías: se puede interactuar con la obra incluso sin que esté físicamente presente · Fundación Mainel </a>
```

Comment:
Add a br here as Spanish version (if you change the zoom or you have a different resolution it will see different as Spanish)